### PR TITLE
Milestone-2: User interface

### PR DESCRIPTION
**Pull request by ajkacca457**
- 2nd time we are creating this pull request because the previous one got accidentally merged. Couldn't revert the changes. 

- all the reviews about this milestone are available at https://github.com/ajkacca457/Project-OOP-TicTacToe/pull/2 request.

- Added basic game flow

- Added sample variable and functions

- Made the pseudo game logic work

- Wrote pseudo-code for all the required future steps

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,2 +1,88 @@
 #!/usr/bin/env ruby
+require_relative '../lib/board'
+require_relative '../lib/helpers'
+
 puts '"Hello world!"'
```
Comment (euqueme) [inline bin/main.rb]: - [x] You don't need to keep this line in this milestone

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @ajkacca457 and @RICKCOYL :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewers, :medal_sports: :medal_sports: your user interface looks good, :ok_hand:  I like the fact that you request for the player's games this makes your game more user friendly :video_game: so please keep it, also that you've used classes for this milestone, let's work on the following changes :construction: :construction_worker: :construction_worker: 

#### Milestone 2
> you don't need to compute who is winning but the game should be able to say something like "Player1 is your turn" instead of just requesting an input.
- [x] `[REQUIRED]` game tells which player turn it is (without computing who the winner is)

### That's it, team :raised_hands: 

### Please :pray: do the changes :technologist: :technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: bin/main.rb (hunk 1 above)
Comment (ajkacca457) [reply bin/main.rb]: Thanks for your review. We have fixed the given issues according to review.

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: :100: 
### HELLO @ajkacca457  VERY NICE WORK AND GREAT PROGRESS SO FAR. GREAT WORK MAKING THE CHANGES AS REQUESTED BY THE PREVIOUS TSE. :clap: 

### ON TO THE NEXT MILESTONE :+1: 
### HAPPY CODING

**Pull request merged**